internal/pkg/component-protocol/issueFilter/conditions.go

```diff
@@ -45,22 +46,22 @@ const (
 )
 
 type FrontendConditions struct {
-	FilterID           string                        `json:"filterID,omitempty"`
-	IterationIDs       []int64                       `json:"iterationIDs,omitempty"`
-	Title              string                        `json:"title,omitempty"`
-	StateBelongs       []apistructs.IssueStateBelong `json:"stateBelongs,omitempty"`
```

Comment:
why change to string type

Reply:
issue related string alias in apistructs will be removed. used protobuf Enum instead.